aspnetcore/blazor/get-started.md

```diff
@@ -15,91 +15,115 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
+   For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
 
-   1\. Install [Visual Studio 2019 version 16.4 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.
+1. Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> to run the app.
 
-   2\. Create a new project.
+> [!NOTE]
+> If you installed the Blazor Visual Studio extension for a prior preview release of ASP.NET Core Blazor, you can uninstall the extension. Installing the Blazor templates in a command shell is now sufficient to surface the templates in Visual Studio when using a version prior to 16.6 Preview 2.
 
-   3\. Select **Blazor App**. Select **Next**.
+# [Visual Studio Code](#tab/visual-studio-code)
 
-   4\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+1. Install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
 
-   5\. For a Blazor WebAssembly experience, choose the **Blazor WebAssembly App** template. For a Blazor Server experience, choose the **Blazor Server App** template. Select **Create**. For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>. If the Blazor WebAssembly template isn't present, return to the previous step and reinstall the template.
+1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template:
 
-   6\. Press **Ctrl**+**F5** to run the app.
+   * Install the [.NET Core 3.1.201 or later (Preview) SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+   * Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
+
+   ```dotnetcli
+   dotnet new -i Microsoft.AspNetCore.Components.WebAssembly.Templates::3.2.0-preview3.20168.3
+   ```
 
    > [!NOTE]
-   > If you installed the Blazor Visual Studio extension for a prior preview release of ASP.NET Core Blazor (Preview 6 or earlier), you can uninstall the extension. Installing the Blazor templates in a command shell is now sufficient to surface the templates in Visual Studio.
+   > .NET Core SDK version 3.1.201 or later is **required** to use the 3.2 Preview 3 Blazor WebAssembly template. Confirm the installed .NET Core SDK version by running `dotnet --version` in a command shell.
 
-   # [Visual Studio Code](#tab/visual-studio-code)
+1. Install [Visual Studio Code](https://code.visualstudio.com/).
 
-   1\. Install [Visual Studio Code](https://code.visualstudio.com/).
+1. Install the latest [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
```

Comment:
I think for VS Code users we should probably go ahead and tell them here to also install the JavaScript Debugger (Nightly) extension and to turn on preview debugging, like we do in the debugging topic. Might as well get them setup right from the start.